.github/CHANGELOG.md

```diff
@@ -0,0 +1,61 @@
+## Release 0.6.0a3 (development release)
```

Comment:
Does the `a3` stand for alpha release number 3? Where's `a1` and `a2`? Is this tag actually necessary? 🤔

Reply:
Yes, that shows our alpha releases. I believe @ilan-tz, myself, and others agreed it is best to consider the current versions as alpha given there will be new versions comping with new and often breaking structures and features. 

`a1` is released, so all good there. I believe `a2` we decided to not release to PyPI (@ilan-tz knows more details) -- either way, since we are at alpha 3, I suggest we continue the path, conforming to PEP440, to our beta and then removing pre-release tags.